src/stacks-hierarchy/RootStack.tsx

```diff
@@ -77,10 +79,16 @@ export const RootStack = () => {
   const {theme} = useTheme();
   const navRef = useNavigationContainerRef<RootStackParamList>();
   useFlipper(navRef);
+  const {userId} = useAuthState();
+  const queryClient = useQueryClient();
 
   useBeaconsState();
   useTestIds();
 
+  useEffect(() => {
+    queryClient.invalidateQueries();
```

Comment:
I have a small question here, should we somehow know that `authStatus === 'authenticated' is not authenticated or logged out before invalidate. It might be that the app enters to this useEffects always when the app restart?

Reply:
Removed it here and now rather doing the invalidation on log out and login both for Vipps and with one time code :) 
@gorandalum or @marius-at-atb would possibly have some input on this solution vs. the previous one with useEffect in RootStack?